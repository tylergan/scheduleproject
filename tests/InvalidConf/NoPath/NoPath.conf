on Tuesday at 1100 run