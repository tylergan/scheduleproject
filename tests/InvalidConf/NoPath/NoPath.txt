Testing that runner.py outputs the correct output to STDERR when there is:
    1) No path given in the command line argument in the config file