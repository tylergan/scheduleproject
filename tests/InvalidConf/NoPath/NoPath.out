error in configuration: on Tuesday at 1100 run
